docs/tutorial/part_2.rst

```diff
@@ -230,12 +233,12 @@ Put the following code in ``chat/routing.py``:
 .. code-block:: python
 
     # chat/routing.py
-    from django.urls import re_path
+    from django.urls import path
 
     from . import consumers
 
     websocket_urlpatterns = [
-        re_path(r'ws/chat/(?P<room_name>\w+)/$', consumers.ChatConsumer),
+        path('ws/chat/<str:room_name>/', consumers.ChatConsumer),
```

Comment:
Did you check this actually works? c.f. #1354

Reply:
Yeah it works for me on django 3.0 and channels 2.4.0, and I tried downgrading to 2.1 and channels 2.2.0 and it worked there too.

Pushed at https://github.com/adamchainz/channels-tutorial

Reply:
OK, but I just haven't merged this because of the report leading to #1354, which had an issue connecting to the `chat/???` route in the next step.

Reply:
Okay I'll revert this bit

Reply:
Hey Adam. Right, sorry for the slow uptake here. I’ve been pondering this. 

I think all and sundry will want to use path here. We’ve already hit this three times. (Fixed once. Broken again. Fixed. Now suggested for a change.)

So. I think we should call it out in the example and link to an open issue here to make it work with path. That way it’s 100% obvious that re_path is required, and maybe someone picks it up and implements it. 

Make sense?

Reply:
#1428 is the issue, updated the tutorial with a note and the URLRouter docs with alink to the issue.